packages/orbit-components/src/Separator/index.jsx

```diff
@@ -1,29 +1,69 @@
 // @flow
 import * as React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 import defaultTheme from "../defaultTheme";
 import getSpacingToken from "../common/getSpacingToken";
+import { left, right } from "../utils/rtl";
 
-import type { Props } from ".";
+import type { Props, Indent } from ".";
+
+function capitalize(string: string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function getIndentAmount({
+  theme,
+  indent,
+}: // StyledContainer takes more props, so this shouldn't be exact
+// eslint-disable-next-line flowtype/require-exact-type
+{
+  theme: typeof defaultTheme,
+  indent: Indent,
+}) {
+  return indent === "none" ? 0 : theme.orbit[`space${capitalize(indent)}`];
+}
+
+const StyledContainer = styled.div`
+  ${({ align }) => css`
+    box-sizing: border-box;
+    width: 100%;
+    ${align === "left" &&
+    css`
+      padding-${right}: ${getIndentAmount};
+    `};
+    ${align === "right" &&
+    css`
+      padding-${left}: ${getIndentAmount};
+    `};
+    ${align === "center" &&
+    css`
+      padding-${left}: ${getIndentAmount};
+      padding-${right}: ${getIndentAmount};
```

Comment:
```suggestion
      padding: 0 ${getIndentAmount};
```

no need for rtl utility here for both sides :)